<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PulseWidthDialog</class>
 <widget class="QDialog" name="PulseWidthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>脉宽</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourcefile.qrc">
    <normaloff>:/Icon/Icon/wave-4.png</normaloff>:/Icon/Icon/wave-4.png</iconset>
  </property>
  <widget class="QFrame" name="frame_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>241</width>
     <height>163</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="PulseWidthPeriodPara_Label">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="PulseWidthPeriodPara_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>700.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PulseWidthPeriodParaUnit_Label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDial" name="PulseWidthPeriodPara_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>268</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="PulseWidthAmp_Label">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="PulseWidthAmp_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="PulseWidthAmpUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>78</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Outline</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="PulseWidthAmp_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>240</width>
     <height>163</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="PulseWidthFreq_Label">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="PulseWidthFreq_Dsb">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="decimals">
           <number>5</number>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="PulseWidthFreqUnit_Cmb">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDial" name="PulseWidthFreq_Dial"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="OutputOn_pushBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>89</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
 
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>OUTPUTON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StandBy_pushBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 1px solid #000000;
    border-radius: 5px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #dedede, stop: 0.5 #434343,
                                   stop: 0.51 #000000, stop: 1 #656a6d);
    color: #FFFFFF;
    font: bold 14px;
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #cfcccc, stop: 0.5 #333232,
                                   stop: 0.51 #000000, stop: 1 #585c5f);
    color: #00CC00;
}
QPushButton:flat {
    border: none;
}</string>
   </property>
   <property name="text">
    <string>STANDBY</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
